pages/docs/concepts/asyncapi-document/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Introduction'
+weight: 1
+---
+
+The AsyncAPI document is a YAML or JSON file that describes your implementation of the AsyncAPI specification. Providing a standardized way to document and describe asynchronous and event-driven systems, the AsyncAPI document generates documentation/code and defines the different components of your event-driven application, such as the events, channels, and message formats.
+
+Primarily, it provides a method to produce comprehensive, consistent, and standardized AsyncAPI documentation. Additionally, it serves as a contract for communication between consumers and producers in an event-driven system. It specifies what should be included in the payload when a service produces a message and provides information to the consumer about the properties in the message. 
```

Comment:
Regarding the words `Producer/Subscriber`, we recently changed it to become `Sender/Receiver`. See https://github.com/asyncapi/spec/pull/934/files#diff-34bca0c91901dffce093114c7d4cac16477410fbca7cabf3a1b888d7dfc3eb4dR41

Reply:
noted, thank you 🙏

Reply:
I still see the words didn't change